test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -69,6 +69,8 @@ namespace integration_framework {
 
     void run();
 
+    // TODO mboldyrev 09.11.2018 IrohaInstance::getIrohaInstance() looks wierd,
+    //      IR-1885              refactoring requested.
```

Comment:
Since you have been touching that file, could you please update license header too?